package.json

```diff
@@ -88,7 +88,7 @@
     "cordova-lib": "^11.0.0",
     "cordova-osx": "github:apache/cordova-osx",
     "cordova-plugin-clipboard": "github:Jigsaw-Code/outline-cordova-plugin-clipboard#v2.0.0",
-    "cordova-plugin-outline": "file:cordova-plugin-outline",
+    "cordova-plugin-outline": "file:src/cordova-plugin-outline",
```

Comment:
Does this just work as is?

Does the apple rsync code still work with this change?
We need to validate that it not only builds, but also that it builds the right thing.

Reply:
Yeah good point, let me verify that.

Reply:
spot check: https://github.com/Jigsaw-Code/outline-client/pull/1467#discussion_r1016997779